docs/contents/components/forms/segmented-control/accessibility.ja.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: SegmentedControl
+description: "`SegmentedControl`は、ユーザーが複数の選択肢の中から1つを選択するための使用されるコンポーネントです。"
+package_name: "@yamada-ui/segmented-control"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`SegmentedControlButton`にアイコンのみを使用する場合は、`SegmentedControlButton`に`aria-label`を設定します。
```

Comment:
The `aria-label` is not needed. Because the `SegmentedControlButton` already contains a `label` element.

Reply:
@hirotomoyamada 

Screen reader does not read `value` of `SegmentedControlButton`.
I think `aria-label` is needed.
```tsx
 <SegmentedControl>
      <SegmentedControlButton value="plus">
        <Plus />
      </SegmentedControlButton>
      <SegmentedControlButton value="minus">
        <Minus />
      </SegmentedControlButton>
    </SegmentedControl>
```

Reply:
My understanding was wrong. It is true that `aria-label` is required when using icons.